lib/node_modules/@stdlib/stats/base/dists/triangular/pdf/benchmark/benchmark.js

```diff
@@ -47,10 +47,10 @@ bench( pkg, function benchmark( b ) {
 	mode = new Float64Array( len );
 
 	for ( i = 0; i < len; i++ ) {
```

Comment:
```suggestion
	mode = new Float64Array( len );
	for ( i = 0; i < len; i++ ) {
```

There should be no empty lines between initializations. This applies throughout the PR.